src/Utilities/ComputeTotalStats.js

```diff
@@ -37,69 +41,313 @@ function computeTotalStats(parts) {
+      totalENCapacity - postRecoveryENSupply,
+      ENRechargeDelayRedline,
+      ENSupplyEfficiency
+    );
+
     return {
+      defensive_performance: average(
+        totalAntiKineticDefense,
+        totalAntiEnergyDefense,
+        totalAntiExplosiveDefense
+      ),
+      /**/
       AP: totalAP,
-      EN_load: totalENLoad,
-      EN_output: ENOutput,
-      EN_capacity: totalENCapacity,
-      total_weight: totalWeight,
-      total_load: totalWeight - legWeight,
-      total_arms_load: totalArmsLoad,
-      load_limit: loadLimit,
-      arms_load_limit: armsLoadLimit,
-      attitude_stability: totalAttitudeStability,
-      attitude_recovery: totalAttitudeRecovery,
       anti_kinetic_defense: totalAntiKineticDefense,
       anti_energy_defense: totalAntiEnergyDefense,
       anti_explosive_defense: totalAntiExplosiveDefense,
-      firearm_spec: firearmSpec,
-      speed: speed,
-      qb_reload_time: QBReloadTime,
+      attitude_stability: totalAttitudeStability,
+      attitude_recovery: computeAttitudeRecovery(totalWeight),
+      /**/
+      target_tracking: getTargetTracking(firearmSpec),
+      /**/
+      speed: computeBoostSpeed(totalWeight, baseSpeed),
+      qb_speed: computeQuickBoostSpeed(totalWeight, baseQBSpeed),
       qb_EN_consumption: QBENConsumption,
-      qb_speed: QBSpeed,
-      booster_efficiency_adj: boosterEfficiencyAdj,
-      generator_supply_adj: generatorSupplyAdjustment,
-      qb_reload_ideal_weight: boosterIdealWeight,
-      EN_recharge: ENRecharge,
-    };
+      qb_reload_time: computeQBReloadTime(baseQBReloadTime, boosterIdealWeight, totalWeight),
+      /**/
+      EN_capacity: totalENCapacity,
+      EN_supply_efficiency: ENSupplyEfficiency,
+      EN_recharge_delay: ENRechargeDelay,
+      /**/
+      total_weight: totalWeight,
+      /**/
+      total_arms_load: totalArmsLoad,
+      arms_load_limit: armsLoadLimit,
+      total_load: totalWeight - legWeight,
+      load_limit: loadLimit,
+      EN_load: totalENLoad,
+      EN_output: ENOutput,
+      /* Advanced Stats */
+      effective_AP_kinetic: effectiveAPKinetic,
+      effective_AP_energy: effectiveAPEnergy,
+      effective_AP_explosive: effectiveAPExplosive,
+      effective_AP_avg: average(
+        effectiveAPKinetic,
+        effectiveAPEnergy,
+        effectiveAPExplosive
+      ),
+      infinite_qb_interval: QBJetDuration + timeToRecoverEnergy(
+        QBENConsumption,
+        ENRechargeDelay,
+        ENSupplyEfficiency
+      ),
+      EN_recharge_delay_redline: ENRechargeDelayRedline,
+      full_recharge_time: fullRechargeTime,
+      full_recharge_time_redline: fullRechargeTimeRedline,
+      EN_recovery_func: energyRecoveryFunc(
+        ENRechargeDelay,
+        0,
+        ENSupplyEfficiency,
+        totalENCapacity
+      ),
+      EN_recovery_func_redline: energyRecoveryFunc(
+        ENRechargeDelayRedline,
+        postRecoveryENSupply,
+        ENSupplyEfficiency,
+        totalENCapacity
+      ),
```

Comment:
`EN_recovery_func` and `EN_recovery_func_redline` are not tenatively shown in MainStatsDisplay.js because they are functions that describe the generator recharge over time. Their purpose is to be plotted in a graph that would look like this (blue for non-redline, red for redline):
![image](https://github.com/user-attachments/assets/3b370e91-42d3-4806-9e2d-0291f60e3822)